x-pack/plugins/spaces/server/lib/space_schema.ts

```diff
@@ -13,6 +13,8 @@ export const spaceSchema = Joi.object({
   description: Joi.string().allow(''),
   initials: Joi.string().max(MAX_SPACE_INITIALS),
   color: Joi.string().regex(/^#[a-z0-9]{6}$/, `6 digit hex color, starting with a #`),
-  disabledFeatures: Joi.array().items(Joi.string()),
+  disabledFeatures: Joi.array()
+    .items(Joi.string())
+    .required(),
```

Comment:
I know it's within our rights to change this, but what do you think about keeping this optional on the public interface, and have us default it to an empty array before saving? Might make upgrade scenarios a bit easier if users have already started scripting this stuff.

Reply:
That's fine, previously, something was erroring out if this wasn't specified, so I added this here.